### ISSUE TITLE
[Bug]: 413 (Request Entity Too Large)

### ISSUE THREAD

**Issue by cesboa** [drakkan/sftpgo#1684]
**Not a bug or support issue. NGINX proxy buffer issue.**
If using http upload disable buffering in NGINX, closing issue.

proxy_request_buffering off;
proxy_buffering off;
client_max_body_size 0;

### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.

### Bug description

I get a "413 (Request Entity Too Large)" while uploading via web. The file is 3GB, but halts around 50%. I can't fint any place there should be a limit on 1-1,5GB while uploading, everything is set to 0 (default). The server (proxmox lxc debian container) has 100G free root, 32GB ram + 32GB swap. There is no issue uploading +100 files at 500MB each or downlading 10GB files, but upload halts with the "413 (Request Entity Too Large)" error on POST. The sFTP has no issue uploading huge files.
As I understand it files are atomized (?) while uploading, so it's not like a max memory issue. Is there a config thing I have overlooked or could it be env ting?
Cheers,

### Steps to reproduce

1. Press "Upload Files"
2. Select >=2GB file
3. Press Save.

### Expected behavior

To upload over 1,5GB via web

### SFTPGo version

v2.6.2

### Data provider

Self hosting / proxmox

### Installation method

Other

### Configuration

https://tteck.github.io/Proxmox/#automation

### What are you using SFTPGo for?

Professional user, 1 person business

**Comment by cesboa** [comment]
I tried clean install of sFTPgo and same issue. While uploading the file is not written to disk (root directory) or ram. So et mus be a temp or cache limit? Is it possible to force sFTPgo to write directly to the user directory. I have tested on multiple file size and it caps out at 1.3GB.

**Comment by cesboa** [comment]
Closed this to early. I thought I found out what went wrong, but just as soon as the problem was gone, it came right back. I did a clean install (Ubuntu 24) on another machine, went through the install guide. The first upload was correct in mode 0 (direct to disk). after logging out and in again the issues came back. upload stalls at 1,3GB. total default installation, only created admin user and user. vanilla everything. This must be a bug!
Solution anyone?
